Unable to generate with new 0.7.0 version

**API Platform version(s) affected**: 0.7.0

**Description**  
Hello, today i tried to generate with this command:
`npx @api-platform/client-generator https://localhost:6789/api . --generator nuxt`
But after some time returned this error

- npm ERR! code ENOENT
- npm ERR! syscall chmod
- npm ERR! path C:\Users\WORK\AppData\Roaming\npm-cache\_npx\14424\node_modules\@api-platform\client-generator\lib\index.js
- npm ERR! errno -4058
- npm ERR! enoent ENOENT: no such file or directory, chmod 'C:\Users\WORK\AppData\Roaming\npm-cache\_npx\14424\node_modules\@api-platform\client-generator\lib\index.js'
- npm ERR! enoent This is related to npm not being able to find a file.
- npm ERR! enoent

After that i tried to generate with specific version(0.6.) and worked as always.
So i think the problem related with this new version.

**How to reproduce**  
`npx @api-platform/client-generator https://localhost:6789/api . --generator nuxt` - an error
`npx @api-platform/client-generator@0.6.0 https://localhost:6789/api . --generator nuxt` - works

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -9,7 +9,7 @@ jobs:
   build:
     strategy:
       matrix:
-        node-version: [14.x, 15.x]
+        node-version: ['14', '15']
 
     runs-on: ubuntu-latest
     steps:

--- a/.github/workflows/npm.yml
+++ b/.github/workflows/npm.yml
@@ -9,12 +9,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Build Node.js 15.x
-      uses: actions/setup-node@v1
+    - uses: actions/setup-node@v1
       with:
-        node-version: '15.x'
+        node-version: '15'
         registry-url: 'https://registry.npmjs.org'
     - run: npm install
+    - run: npm build
     - run: npm publish
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}

